core/include/kernel/tpm.h

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/*
+ * Copyright (c) 2020, ARM Limited.
+ */
+
+#ifndef __KERNEL_TPM_H__
+#define __KERNEL_TPM_H__
+
+#include <tee_api_types.h>
+
+#ifdef CFG_CORE_TPM_EVENT_LOG
+
+/*
+ * Returns the TPM Event Log information previously retrieved
+ * by @map_tpm_log_area.
+ *
+ * @buf Pointer to a buffer where to store a copy of the TPM Event log.
+ * @size When this function is called, size contains the size of buf.
+ * Upon returning it contains the actual size of the TPM Event log.
+ *
+ * This function will return TEE_SUCCESS if no errors are detected.
+ */
+TEE_Result tpm_get_event_log(void *buf, size_t *size);
+
+/*
+ * Read the TPM Event log information and store it internally.
+ * It first tries to find the log parameters in a DTB. if none is provided,
+ * it will use the constant parameters hardcoded in conf.mk.
+ * If the TPM Event Log information is properly retrieved,
+ * it will map its address to the OPTEE Core.
+ *
+ * @fdt Pointer to the DTB blob where the TPM Event log information
+ * is expected to be.
+ */
+void tpm_map_log_area(void *fdt);
+
+#else
+
+static inline TEE_Result tpm_get_event_log(void *buf __unused,
+					   size_t *size __unused)
+{
+	return TEE_ERROR_NOT_SUPPORTED;
+}
+
+static inline void tpm_map_log_area(void *fdt __unused)
+{}
+
+#endif				/* __KERNEL_TPM_H__ */
```

Comment:
This `#endif` matches the `#ifdef CFG_CORE_TPM_EVENT_LOG` above.
Btw, I don't think the comment need to be indented at all.